setup.py

```diff
@@ -61,6 +61,7 @@ def find_long_description():
         'graphql-core==1.1',
         'pytz>=2016.10',
         'six>=1.10.0',
+        'sqlalchemy>=1.2.1',
```

Comment:
How do you feel about a `<1.3` or `<2` bound in addition to this? `graphql-core` is pinned to an exact version since it's very important, and I'd imagine that a major (and maybe even minor) `sqlalchemy` version bump might break things as well.

Reply:
Great idea, I'll pin to <1.3 to be safe.